elements/pfe-tabs/BaseTab.ts

```diff
@@ -67,10 +69,10 @@ export abstract class BaseTab extends LitElement {
 
   #activeChanged() {
     if (this.active && !this.disabled) {
-      this.removeAttribute('tabindex');
+      this._button.removeAttribute('tabindex');
```

Comment:
Could we just do this is the lit template? Imperatively modifying DOM that lit controls is fragile

Reply:
PTAL @zeroedin

Reply:
Fixed in https://github.com/patternfly/patternfly-elements/pull/2310/commits/9ac7e3d8fe6c8a2fc81e16c9c9e51695cad4b067